po/tr.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/devel.rs:193
+msgid "failed to save devel info"
+msgstr "devel bilgisi kayıt edilemedi"
+
+#: src/devel.rs:382
+msgid "package does not have an update"
+msgstr "pakete ait güncelleme yok"
+
+#: src/devel.rs:417
+msgid "failed to lookup: {}"
+msgstr "bakılamadı: {}"
+
+#: src/devel.rs:445
+msgid "invalid json: {}"
+msgstr "geçersiz json : {}"
+
+#: src/download.rs:83
+msgid "packages not in the AUR: "
+msgstr "aur'da olmayan paketler: "
+
+#: src/download.rs:94
+msgid "marked out of date: "
+msgstr "tarihi geçmişler: "
+
+#: src/download.rs:105
+msgid "orphans: "
+msgstr "yetimler: "
+
+#: src/download.rs:206
+msgid "could not get current directory"
+msgstr "geçerli dizin alınamadı"
+
+#: src/download.rs:210
+msgid "can not get repo packages: asp is not installed"
+msgstr "depo paketleri alınamadı: asp yüklü değil"
+
+#: src/download.rs:231
+msgid "Missing ABS packages "
+msgstr "Eksik ABS paketleri "
+
+#: src/download.rs:250 src/download.rs:276 src/download.rs:530
+#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
+#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
+#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
+#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+msgid "failed to run:"
+msgstr "çalıştırılamadı:"
+
+#: src/download.rs:263
+msgid "does not contain PKGBUILD: skipping"
+msgstr "pkgbuild içermiyor: es geçiliyor"
+
+#: src/download.rs:294
+msgid " ({:total>padding$}/{:total}) downloading: {:pkg}"
+msgstr " ({:total>padding$}/{:total}) indiliriliyor: {:pkg}"
+
+#: src/download.rs:317
+msgid "Downloading PKGBUILDs..."
+msgstr "PKGBUILD'leri İndiriliyor..."
+
+#: src/download.rs:321
+msgid " PKGBUILDs up to date"
+msgstr " PLGBUILD'leri güncel"
+
+#: src/download.rs:520
+msgid "{} is not installed: can not get repo packages"
+msgstr "{} yüklü değil: depo paketleri alınamadı"
+
+#: src/download.rs:561
+msgid "asp returned {}"
+msgstr "asp cevabı {}"
+
+#: src/exec.rs:86
+msgid "Pacman is currently in use, please wait..."
+msgstr "Pacman şuan kullanılıyor, lütfen bekleyiniz..."
+
+#: src/fmt.rs:6 src/info.rs:146 src/search.rs:176
+msgid "None"
+msgstr "Hiç biri"
+
+#: src/info.rs:30 src/query.rs:51
+msgid "package '{}' was not found"
+msgstr "'{}' paketi bulunamadı"
+
+#: src/info.rs:57 src/info.rs:96
+msgid "Name"
+msgstr "İsim"
+
+#: src/info.rs:58 src/info.rs:97
+msgid "Version"
+msgstr "Versiyon"
+
+#: src/info.rs:59 src/info.rs:98
+msgid "Description"
+msgstr "Açıklama"
+
+#: src/info.rs:60 src/info.rs:106
+msgid "Groups"
+msgstr "Gruplar"
+
+#: src/info.rs:61 src/info.rs:107
+msgid "Licenses"
+msgstr "Lisanslar"
+
+#: src/info.rs:62 src/info.rs:108
+msgid "Provides"
+msgstr "Sağladıkları"
+
+#: src/info.rs:63 src/info.rs:109
+msgid "Depends On"
+msgstr "Gereksinimleri"
+
+#: src/info.rs:64 src/info.rs:110
+msgid "Make Deps"
+msgstr "Make Gereksinimleri"
+
+#: src/info.rs:65 src/info.rs:111
+msgid "Check Deps"
+msgstr "Gereksinimleri Kontrol Et"
```

Comment:
Check Deps = dependencies that needs to be checked, not "check the dependencies"

https://github.com/Morganamilo/paru/blob/5be43f50d934f1045d631e6745866885da4867b3/src/info.rs#L112

It can be translated as `"Kontrol Gereksinimleri"`.